src/components/NavLinks/NavLinks.tsx

```diff
@@ -1,61 +1,117 @@
 import { FC } from 'react';
-import routes from '@/config/routes';
-import { Link, useLocation, useNavigate } from 'react-router-dom';
+import { Route, NavLinksProps } from '@/types/props';
+import NavLink from './NavLink';
 import Paths from '@/config/paths';
-import Button from '../Button/Button';
-import { useMediaQuery } from 'react-responsive';
-import { breakpoints } from '@utils/globals';
-import { NavLinksProps } from '@/types/props';
-import LogoGrey from '@assets/logo/LogoGrey';
+import routes from '@/config/routes';
 import styles from './NavLinks.module.scss';
 
-const NavLinks: FC<NavLinksProps> = ({
-  theme,
-  activeClassName,
-  linkClassName,
-  className,
-  buttonClassName,
-}) => {
-  const isMobile = useMediaQuery({ query: `(max-width: ${breakpoints.screenMedium})` });
-  const navigate = useNavigate();
-  const { pathname } = useLocation();
+interface NavRoute {
+  path: Paths;
+  childNavRoutes?: NavRoute[];
+}
+
+const navRoutes: NavRoute[] = [
```

Comment:
Would this array be better stored in the config folder with the routes files and then import here?

Reply:
Moved to config/navRoutes